.buildkite/scripts/steps/build_and_test.sh

```diff
@@ -32,14 +32,31 @@ if [[ -n "$BUILDKITE_PULL_REQUEST" && -n "$VERSION_QUALIFIER" ]] ; then
     exit 2
 fi
 
-# Tests must be run in PR builds
-if [[ -n "$BUILDKITE_PULL_REQUEST" && "$RUN_TESTS" = false ]] ; then
-    echo "RUN_TESTS should not be false PR builds"
-    exit 3
-fi
+VERSION=$(cat ${REPO_ROOT}/gradle.properties | grep '^elasticsearchVersion' | awk -F= '{ print $2 }' | xargs echo)
+HARDWARE_ARCH=$(uname -m | sed 's/arm64/aarch64/')
 
 echo "environment variables:"
 env
+
+if [[ "$HARDWARE_ARCH" = aarch64 && -z "$CPP_CROSS_COMPILE" ]] ; then 
+  # On Linux native aarch64 build using Docker
+  
+  # The Docker version is helpful to identify version-specific Docker bugs
+  docker --version
+  
+  KERNEL_VERSION=`uname -r`
+  GLIBC_VERSION=`ldconfig --version | head -1 | sed 's/ldconfig//'`
+
+  if [ "$RUN_TESTS" = false ] ; then
+    ${REPO_ROOT}/dev-tools/docker_build.sh linux_aarch64_native
+  else
+    ${REPO_ROOT}/dev-tools/docker_test.sh --extract-unit-tests linux_aarch64_native
+    echo "Re-running seccomp unit tests outside of Docker container - kernel: $KERNEL_VERSION glibc: $GLIBC_VERSION"
+    (cd ${REPO_ROOT}/cmake-build-docker/test/lib/seccomp/unittest && LD_LIBRARY_PATH=`cd ../../../../../build/distribution/platform/linux-aarch64/lib && pwd` ./ml_test_seccomp)
```

Comment:
Since this is running on a VM, we can run the ES integration tests on the same VM.

Just like we did in Jenkins in https://github.com/elastic/ml-cpp/blob/976a2a293ec815ea1b869fd61d243ff1dab30eb5/dev-tools/jenkins_ci.sh#L151-L153

That will mean we need a JVM installed on the VM though.  Eventually that should be achieved by using the same VM image as Elasticsearch, but for now maybe the appropriate `yum` command (or whatever package manager Amazon Linux uses) could be added to the `commands` section of `pipeline.yml`.